src/components/Home/Banner/index.tsx

```diff
@@ -23,6 +24,13 @@ const Banner: FC<IBanner> = ({
   const [showOrgModal, setShowOrgModal] = useState(false);
   const [showPrjModal, setShowPrjModal] = useState(false);
 
+  useEffect(() => {
+    removeItem('file');
+    removeItem('name');
+    removeItem('orgData');
+    removeItem('choices');
+  }, []);
```

Comment:
I think it should only be if the user is coming from signup, better to remove it at the ending stage of signup.